packages/e2e-test-utils/src/insertWidget.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+async function insertWidget(name) {
+  // Hotfix for WordPress 5.3 where `wpWidgets.l10n.widgetAdded`
```

Comment:
We don't test against WordPress 5.3 anymore, so I don't think we need this hotfix.

Reply:
This is just a cut and paste from elsewhere. I can remove this, but it is unrelated to this change.

Reply:
A great opportunity to fix it while we're at it.

Reply:
Removed